packages/compass-generative-ai/src/components/ai-optin-modal.tsx

```diff
@@ -58,29 +51,70 @@ const bannerStyles = css({
   marginTop: spacing[400],
   textAlign: 'left',
 });
-const getButtonText = (isOptInInProgress: boolean) => {
-  return (
-    <>
-      &nbsp;Use Natural Language
-      {isOptInInProgress && (
-        <>
-          &nbsp;
-          <SpinLoader darkMode={true}></SpinLoader>
-        </>
-      )}
-    </>
-  );
+
+// TODO: The LG MarketingModal does not provide a way to disable the button
+// so this is a temporary workaround to make the button look disabled.
+const focusSelector = `&:focus-visible, &[data-focus="true"]`;
+const hoverSelector = `&:hover, &[data-hover="true"]`;
+const activeSelector = `&:active, &[data-active="true"]`;
+const focusBoxShadow = (color: string) => `
+    0 0 0 2px ${color}, 
+    0 0 0 4px ${palette.blue.light1};
+`;
+const disabledButtonStyles: Record<Theme, string> = {
+  [Theme.Light]: css`
+    button {
```

Comment:
This seems to affect the close button also:

<img width="400" height="755" alt="image" src="https://github.com/user-attachments/assets/d66e384f-c719-4357-901c-aea7b103a0eb" />

And this is how it looks when these are not applied to modal:

<img width="400" height="759" alt="image" src="https://github.com/user-attachments/assets/8b0683df-0d0d-4d70-84e6-68216efc4e25" />

Is there a more precise selector you can use here?

Reply:
Changed the selector to `'button[data-lgid="lg-button"]:not([aria-label="Close modal"])`.